### ISSUE TITLE
Knowledge graph appears to be created, search engine says 200 documents loaded, but nothing shows in the search query

### ISSUE THREAD

**Issue by cslovell** [usc-isi-i2/dig-etl-engine#267]
Is this a known issue? I've got some fields added, glossaries, and everything seems fine--except when I go to the search UI, nothing shows. Any tips?

**Comment by cslovell** [comment]
This is what I mean--everything from the MyDIG UI seems fine, but the search is blank. Landmark generated the right table and was "published". 
![search_blank](https://user-images.githubusercontent.com/6457695/46020601-b9dd5c00-c0ac-11e8-9803-32f804382dde.png)
![mydiglooksfine](https://user-images.githubusercontent.com/6457695/46020609-bf3aa680-c0ac-11e8-81ca-4bdcd4ac62ff.png)

**Comment by GreatYYX** [comment]
https://github.com/usc-isi-i2/dig-etl-engine/issues/265#issuecomment-423293507

@saggu would be better to add this explanation to digui instruction

**Comment by saggu** [comment]
@GreatYYX yeah documentation needs update. @cslovell please see this comment : https://github.com/usc-isi-i2/dig-etl-engine/issues/265#issuecomment-423612466

Basically in myDIG, you have to set whichever fields you want to be in the search form

**Comment by sobalgi** [comment]
@cslovell Are you able to search text populated in the fields. [#265 (comment)](https://github.com/usc-isi-i2/dig-etl-engine/issues/265#issuecomment-423617587)
